src/RightsService.Models.Dto/Responses/RoleResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿using LT.DigitalOffice.RightsService.Models.Dto.Models;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.RightsService.Models.Dto.Responses
+{
+    public class RoleResponse
```

Comment:
We have find response and role response, for what?

Reply:
done